### PR TITLE
{lib}[GCCcore/10.2.0] Blitz++ v1.0.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+postinstallcmds = ['mkdir %(installdir)s/lib64/blitz']
+
+sanity_check_paths = {
+    'files': ['lib64/libblitz.a', 'lib64/libblitz.%s' % SHLIB_EXT],
+    'dirs': ['include/blitz/array', 'include/blitz/meta',
+             'include/random', 'lib64/pkgconfig'],
+}
+
+configopts = '--enable-shared'
```
Comment (Micket) [inline easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb:33]: sorry for nitpick, but can we have configopts directly below dependencies?
Also, did we need this? Previous easyconfig, same version, it seems to be default?

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+source_urls = [('https://github.com/blitzpp/blitz/archive/')]
+sources = ['%(version)s.tar.gz']
+checksums = ['500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2']
+
+builddependencies = [('CMake', '3.18.4'), ('binutils', '2.35')]
+
+separate_build_dir = True
+
+postinstallcmds = ['mkdir %(installdir)s/lib64/blitz']
```
Comment (Micket) [inline easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb:25]: I don't see how this is necessary? It's post install, so, it just creates an empty directory?

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb]: I think this is left over the ConfigureMake variant of this EB file so I've now removed it. Thanks for spotting this!

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb]: I suspect this left over from some other tool I was trying to build that looked for this folder and complained when it didn't exist.  I've removed it now.

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [('https://github.com/blitzpp/blitz/archive/')]
+sources = ['%(version)s.tar.gz']
+checksums = ['500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2']
+
+builddependencies = [('CMake', '3.18.4'), ('binutils', '2.35')]
+
+separate_build_dir = True
```
Comment (Micket) [inline easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb:23]: drop this as well in both PRs. separate_build_dir is True by default now

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb (hunk 3 above)
Comment (Micket) [reply easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb:23]: just the separate_build_dir , the review looks like i'm complaining about builddeps as well (I'm not!)

Comment (SimonPinches) [reply easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-GCCcore-10.2.0.eb]: Okay, I've dropped the separate_build_dir from both as well now.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.5, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/46f40760ea7ea7cdd439edb3cdac20a3 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**